cmd_gen_finetuning.ipynb

```diff
@@ -154,18 +162,26 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "from unsloth import FastLanguageModel\n",
     "\n",
     "# adapt model for peft\n",
     "model = FastLanguageModel.get_peft_model(\n",
     "    model,\n",
-    "    r=32,\n",
-    "    lora_alpha=16,\n",
-    "    lora_dropout=0.05,\n",
-    "    use_gradient_checkpointing=\"unsloth\",  # additional optimisation\n",
+    "    r = 16,\n",
+    "    target_modules = [\"q_proj\", \"k_proj\", \"v_proj\", \"o_proj\",\n",
+    "                      \"gate_proj\", \"up_proj\", \"down_proj\",],\n",
+    "    lora_alpha = 16,\n",
+    "    lora_dropout = 0,\n",
+    "    bias = \"none\",\n",
+    "    use_gradient_checkpointing = \"unsloth\",\n",
+    "    random_state = random_seed,\n",
+    "    use_rslora = False,\n",
+    "    loftq_config = None,\n",
```

Comment:
Why set `use_rslora = False` and `loftq_config = None`?

Reply:
same reasoning

Reply:
@ rafaeljcdarce did you test the config's fine-tuning performance on some data/bot?